dali/daliadmin/daliadmin.cpp

```diff
@@ -129,6 +129,7 @@ void usage(const char *exe)
   printf("  wuidcompress <wildcard> <type>  --  scan workunits that match <wildcard> and compress resources of <type>\n");
   printf("  wuiddecompress <wildcard> <type> --  scan workunits that match <wildcard> and decompress resources of <type>\n");
   printf("  xmlsize <filename> [<percentage>] --  analyse size usage in xml file, display individual items above 'percentage' \n");
+  printf("  migratefiles <src-group> <target-group> [<filemask>] [DRYRUN] [CREATEMAPS] [LISTONLY] [VERBOSE]\n");
```

Comment:
could change usage print, but tests are case-insensitive, so no need for conversion.